samples/bluetooth/gattc_read_write/README.md

```diff
@@ -0,0 +1,74 @@
+Bluetooth Client Read and Write Sample Application
+==================================================
+
+This example demonstrates usage of the XBee's BLE client read/write feature.
+The example connects to a Silicon Labs Thunderboard and reads its temperature.
+The example also toggles the LED every couple seconds.
+
+Requirements
+------------
+
+To run this example you need:
+
+* One XBee3 radio module with MicroPython support.
+* One carrier board for the radio module (XBIB-U-DEV or XBIB-C board).
+* A Silicon Labs [Thunderboard React](https://www.silabs.com/products/development-tools/thunderboard/thunderboard-react-kit-sensor-cloud-connectivity)
+
+Setup
+-----
+
+Make sure the hardware is set up correctly:
+
+1. Plug the XBee3 radio module into the XBee adapter and connect it to your
+   computer's USB port.
+2. Turn on the Thunderboard and press button SW-0 to make it connectable by
+   pressing the SW-0 button. LED-B will begin to blink indicating that the
+   Thunderboard is ready for a connection.
+
+Run
+---
+
+The example will need to be configured, Enter the BLE address of the
+Thunderboard React board. The sample will make a connection to the
```

Comment:
s/React board/React/  (seems redundant to have "Thunderboard" and "board").

Also try to remove use of "will" in the documentation.  "The sample makes a connection" reads easier than "The sample will make a connection".